mollie-payments-for-woocommerce/includes/mollie/wc/plugin.php

```diff
@@ -216,12 +217,38 @@ public static function init() {
 		// Capture order at Mollie (for Orders API/Klarna)
 		add_action( 'woocommerce_order_status_completed', array( __CLASS__, 'shipAndCaptureOrderAtMollie' ) );
 
+        // Enqueue Scripts
+        add_action('wp_enqueue_scripts', [__CLASS__, 'enqueueFrontendScripts']);
+
 		self::initDb();
 		self::schedulePendingPaymentOrdersExpirationCheck();
 		// Mark plugin initiated
 		self::$initiated = true;
 	}
 
+    /**
+     * Enqueue Frontend only scripts
+     *
+     * @return void
+     */
+    public static function enqueueFrontendScripts()
```

Comment:
Why `public static`? `static` prevents any kind of injection, and `public` means that anything can invoke this method. Does this method need to be part of the interface, i.e. invoked from outside of this class?

Reply:
@XedinUnknown Have a look at all of the other methods, them are static, I want to keep it the same. May be in future we can improve it.